src/ios/RudderSDKCordovaPlugin.m

```diff
@@ -175,4 +176,16 @@ - (void)setAnonymousId:(CDVInvokedUrlCommand*)command
     [RSClient setAnonymousId:anonymousId];
 }
 
++ (void) addFactory:(id<RSIntegrationFactory>) factory 
+{
+    if (factories == nil) {
+        factories = [[NSMutableArray alloc] init];
```

Comment:
@desusai7 can you initialize this array at this class's initialize method?

Reply:
Yeah Sure